project/AppDependencies.scala

```diff
@@ -6,34 +6,34 @@ object AppDependencies {
 
   val compile = Seq(
     "commons-validator" % "commons-validator" % "1.7",
-    "uk.gov.hmrc"   %% "auth-client"                % "5.6.0-play-27",
-    "uk.gov.hmrc"   %% "play-frontend-hmrc"         % "0.79.0-play-27",
-    "uk.gov.hmrc"   %% "bootstrap-frontend-play-27" % "5.3.0",
-    "uk.gov.hmrc"   %% "govuk-template"             % "5.68.0-play-27",
-    "uk.gov.hmrc"   %% "http-caching-client"        % "9.5.0-play-27",
-    "uk.gov.hmrc"   %% "mongo-lock"                 % "7.0.0-play-27",
-    "uk.gov.hmrc"   %% "play-health"                % "3.16.0-play-27",
-    "uk.gov.hmrc"   %% "play-language"              % "5.1.0-play-27",
-    "uk.gov.hmrc"   %% "simple-reactivemongo"       % "8.0.0-play-27",
-    "uk.gov.hmrc"   %% "play-allowlist-filter"      % "1.0.0-play-27",
-    "uk.gov.hmrc"   %% "play-json-union-formatter"  % "1.13.0-play-27",
-    "org.typelevel" %% "cats-core"                  % "2.2.0"
+    "uk.gov.hmrc"   %% "play-frontend-hmrc"         % "0.94.0-play-28",
+    "uk.gov.hmrc"   %% "bootstrap-frontend-play-28" % "5.7.0",
```

Comment:
upgrade bootstrap version to 5.14.0
upgrade play-frontend-hmrc  to 1.9.0
remove govuk-template library